src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,76 @@
+package calculator;
+
+public class Calculator {
+
+    public String calculate(String inputString) {
+
+        String[] inputs = splitInputString(inputString);
+
+        FourPointOperation operatorOrNumber = FourPointOperation.NUMBER;
+        Integer result = Integer.parseInt(inputs[0]);
+
+        for (String input : inputs) {
+            Expression expression = new Expression(result, operatorOrNumber, input);
+            result = calculateFor(expression);
+            operatorOrNumber = getFourPointOperation(input);
+        }
+
+        return result.toString();
+    }
+
+    private String[] splitInputString(String inputString) {
+
+        if (inputString == null) {
```

Comment:
```suggestion
        if (Obejcts.isNull(inputString)) {
```
Objects 클래스에 대해 알아보고 활용해보세요 !